api/apps/api/test/scenarios/input-file.e2e-spec.ts

```diff
@@ -203,6 +230,16 @@ async function getFixtures() {
         .set('Authorization', `Bearer ${jwtToken}`)
         .send(input);
     },
+    async ThenScenarioHasMetadataForMarxan(id: string) {
+      return await request(app.getHttpServer())
+        .get(`/api/v1/marxan-run/scenarios/${id}/marxan/dat/input.dat`)
+        .set(
+          'X-Api-Key',
+          process.env.API_AUTH_X_API_KEY ?? 'sure it is valid in envs?',
```

Comment:
can we get this via `config`?

Reply:
so that we only couple setting to env var in `custom-environment-variables.json`

Reply:
not sure if this is desirable from e2e point of view, no hard opinion tho.

@Dyostiq your take?

Reply:
I have no hard opinion either - I would say that having a way to refer to settings consistently throughout may be desirable, but if the coupling was updated and we forgot to update the env var here the test would fail so we'd notice (but then extra tedious work 😅)

Reply:
if it's e2e, by using env we test a bit more, like if the mapping is correct etc.
I try to have e2e tests as much separate from the production code and tools as possible – then, they're still valid after refactor.